CODE_OF_CONDUCT.md

```diff
@@ -1,75 +1,48 @@
-  address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+* The use of sexualized language or imagery, and sexual attention or advances of any kind. 🚫❌🙅‍♀️
+* Trolling, insulting, or derogatory comments, and personal or political attacks. 🚫😡🙅‍♂️
+* Public or private harassment. 🚫😔😖
+* Publishing others' private information, such as a physical or email address, without their explicit permission. 🚫📤📧
+* Other conduct which could reasonably be considered inappropriate in a professional setting. 🚫👎💼
 
-## Enforcement Responsibilities
+## 👮‍♂️ Enforcement Responsibilities
 
-Community leaders are responsible for clarifying and enforcing our standards of
-acceptable behavior and will take appropriate and fair corrective action in
-response to any behavior that they deem inappropriate, threatening, offensive,
-or harmful.
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior. They will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
 
-Community leaders have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned to this Code of Conduct, and will communicate reasons for moderation
-decisions when appropriate.
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct. They will communicate reasons for moderation decisions when appropriate.
 
-## Scope
+## 📚 Scope
 
-This Code of Conduct applies within all community spaces, and also applies when
-an individual is officially representing the community in public spaces.
-Examples of representing our community include using an official e-mail address,
-posting via an official social media account, or acting as an appointed
-representative at an online or offline event.
+This Code of Conduct applies within all community spaces and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
 
-## Enforcement
+## 🔒 Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-gmail.
-All complaints will be reviewed and investigated promptly and fairly.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at [insert email address here]. All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
Change this line - [insert email address here]

Reply:
ok

> Change this line - [insert email address here]